src/sentry/static/sentry/app/views/alerts/rules/details/relatedIssues.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import {SectionHeading} from 'app/components/charts/styles';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import GroupList from 'app/components/issues/groupList';
+import {Panel, PanelBody} from 'app/components/panels';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {OrganizationSummary} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {IncidentRule} from 'app/views/settings/incidentRules/types';
+
+type Props = {
+  organization: OrganizationSummary;
+  rule: IncidentRule;
+  filter: string;
+  statsPeriod?: string;
+  start?: string;
+  end?: string;
+};
+
+class RelatedIssues extends React.Component<Props> {
+  renderEmptyMessage = () => {
+    return (
+      <Panel>
+        <PanelBody>
+          <EmptyStateWarning small withIcon={false}>
+            {t('No issues for this alert rule')}
+          </EmptyStateWarning>
+        </PanelBody>
+      </Panel>
+    );
+  };
+
+  render() {
+    const {rule, filter, organization, start, end, statsPeriod} = this.props;
+
+    const path = `/organizations/${organization.slug}/issues/`;
+    const queryParams = {
+      start,
+      end,
+      statsPeriod,
+      limit: 5,
+      sort: 'new',
+      query: `${rule.query} ${filter}`,
```

Comment:
we might need to send project down here because I ran into this error

![image](https://user-images.githubusercontent.com/9372512/105541650-e9b4c280-5cac-11eb-83f9-3e9423c6b486.png)

^ due to not having multi-project streaming enabled, but each metric alert only has one project so we should just fetch the issues of that project

Reply:
You're right